src/main/java/tr/ogu/edu/school/schoolapp/controller/ActTicketController.java

```diff
@@ -38,4 +39,14 @@ public ResponseEntity<Resource> getTicketAsPdf(@PathVariable String verification
 				.header(HttpHeaders.CONTENT_DISPOSITION, "inline;filename=" + verificationCode + ".pdf").body(resource);
 	}
 
-}
\ No newline at end of file
+	// Biletin kullanım durumu
+	@PostMapping("/public/{verificationCode}/{line}/{no}")
+	public ResponseEntity<Map<String, Object>> verifyTicket(@PathVariable String verificationCode,
+        	@PathVariable String line, @PathVariable String no) {
+
+    	// Biletin durumunu doğrular ve günceller
+    	String qrCode = verificationCode;
```

Comment:
Bu satıra gerek var mı?

Reply:
Doğrulama kodunun qr olduğunu belirtir diye yazmıştım

Reply:
Backend olarak burada QR kodu mu barcode mu yoksa sadece düz bir metin mi bizi ilgilendirmez. Bir doğrulama kodu geliyor ve bununla ilgili süreci işletiyorsun. Bunu QRCode olarak okumak ve sonrasında bize okuduğu değeri düz metin olarak göndermek frontend'in sorumluluğunda.